pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -239,6 +242,25 @@ func (c Cloud) deleteS3Objects(b string, keys []string) error {
 	return err
 }
 
+func (c Cloud) getS3Object(bucket, objectName string) ([]byte, error) {
+	params := &s3.GetObjectInput{
+		Bucket: aws.String(bucket),
+		Key:    aws.String(objectName),
+	}
+
+	resp, err := c.s3.GetObject(params)
+	if err != nil {
+		return []byte{}, err
```

Comment:
Maybe return `nil` instead of an empty byte array (when this fails)? (and likewise below)